packages/itwin/tree-widget/src/e2e-tests/ModelsTree.test.ts

```diff
@@ -57,13 +60,16 @@ test.describe("Models tree", () => {
 
   test("expanded tree node", async ({ page }) => {
     const physicalModelNode = locateNode(treeWidget, "ProcessPhysicalModel");
-    await physicalModelNode.getByRole("button", { name: "Toggle", includeHidden: true }).click();
+    await physicalModelNode.click();
+    await page.keyboard.press("ArrowRight");
 
     const pipeSupportNode = locateNode(treeWidget, "PipeSupport");
-    await pipeSupportNode.getByRole("button", { name: "Toggle", includeHidden: true }).click();
+    await pipeSupportNode.click();
+    await page.keyboard.press("ArrowRight");
```

Comment:
Can this be extracted into utility function like `expandTreeNode()`?

Reply:
Good call, added utility function and changed to use focus instead of click.